tests/Bridge/Doctrine/EventListener/WriteListenerTest.php

```diff
@@ -108,20 +108,23 @@ public function testOnKernelViewWithNoResourceClass()
         $dummy = new Dummy();
         $dummy->setName('Dummyrino');
 
+        $objectManagerProphecy = $this->prophesize(ObjectManager::class);
+        $objectManagerProphecy->flush()->shouldNotBeCalled();
         $managerRegistryProphecy = $this->prophesize(ManagerRegistry::class);
+        $managerRegistryProphecy->getManagerForClass('Dummy')->willReturn($objectManagerProphecy);
 
         $writeListener = new WriteListener($managerRegistryProphecy->reveal());
         $httpKernelProphecy = $this->prophesize(HttpKernelInterface::class);
         $request = new Request();
         $request->setMethod('POST');
         $event = new GetResponseForControllerResultEvent($httpKernelProphecy->reveal(), $request, HttpKernelInterface::MASTER_REQUEST, $dummy);
 
-        $this->assertNull($writeListener->onKernelView($event));
-        $this->assertNotEquals($dummy, $writeListener->onKernelView($event));
+        $writeListener->onKernelView($event);
     }
 
     /**
      * @expectedDeprecation The ApiPlatform\Core\Bridge\Doctrine\EventListener\WriteListener class is deprecated since version 2.2 and will be removed in 3.0. Use the ApiPlatform\Core\EventListener\WriteListener class instead.
+     * @doesNotPerformAssertions
```

Comment:
Isn't this missing a `@group legacy` instead ?

Reply:
The `@group legacy` is already on the class. And `@doesNotPerformAssertions` is not related to that.